Content.Client/Audio/ClientAdminSoundSystem.cs

```diff
@@ -9,14 +9,16 @@ namespace Content.Client.Audio;
 public sealed class ClientAdminSoundSystem : SharedAdminSoundSystem
 {
     [Dependency] private readonly IConfigurationManager _cfg = default!;
-
+    
+    private bool _adminAudioEnabled = true;
     private List<IPlayingAudioStream?> _adminAudio = new(1);
 
     public override void Initialize()
     {
         base.Initialize();
         SubscribeNetworkEvent<AdminSoundEvent>(PlayAdminSound);
         _cfg.OnValueChanged(CCVars.AdminSoundsEnabled, ToggleAdminSound, true);
+        _adminAudioEnabled = _cfg.GetCVar(CCVars.AdminSoundsEnabled);
```

Comment:
```suggestion
```
No need to get the CVar here as `OnValueChanged` gets called immediately, thanks to the third argument.